geode-core/src/main/java/org/apache/geode/cache/AttributesMutator.java

```diff
@@ -157,6 +157,14 @@
   CacheLoader<K, V> setCacheLoader(CacheLoader<K, V> cacheLoader);
 
 
+  /**
+   * Updates Partitioned Region Configuration for the cache loader and writer
+   *
+   * @param cacheLoader the cache loader after update
+   * @param cacheLoader the cache writer after update
+   */
+  void updatePRNodeInformation(CacheLoader<K, V> cacheLoader, CacheWriter<K, V> cacheWriter);
```

Comment:
This was a change we debated on, eventually moving it to attributes mutator because we thought it made more conceptual sense as functionality of that class. I think given what you've pointed out here, it makes more sense to move the definition back into the derived class, so we can do that.